static/src/stylesheets/module/atoms/_storyquestions.scss

```diff
@@ -0,0 +1,139 @@
+.user__question {
+    padding-bottom: $gs-baseline;
+    overflow: auto;
+}
+.user__question-title {
+    @include fs-headline(3);
+    font-weight: 900;
+    color: $news-main-1;
+
+    @include mq($from: tablet) {
+        @include fs-headline(4);
+        font-weight: 900;
+    }
+}
+
+.user__question-title--secondary {
+    color: $neutral-1;
+    display: block;
+}
+
+.user__questions-text--wrapper {
+    position: relative;
+    overflow: hidden;
+
+    &:hover {
+        .user__question-text {
+            color: #ffffff;
+        }
+        .inline-tick {
+            background-color: $news-main-1;
+        }
+        .user__questions-text--inner {
+            background-color: $news-main-1;
+        }
+        .inline-tick__svg {
+            path {
+                fill: #ffffff;
+            }
+        }
+    }
+}
+
+.user__question-section {
+    border-top: 1px dotted $neutral-6;
+    margin-top: $gs-baseline * 2;
+
+    h3 {
+        @include fs-textSans(2);
+        color: $neutral-2;
+        font-weight: 200;
+    }
+}
+
+.user__question-text {
+    @include fs-headline(2);
+    font-weight: 900;
+    display: block;
+    max-width: 68%;
+    // font-size: 18px;
+    float: left;
+    color: $news-main-1;
+    padding-left: $gs-baseline / 4;
+    position: relative;
+    margin-bottom: $gs-baseline;
+
+    @include mq($from: tablet) {
+        max-width: 80%;
+        @include fs-headline(3);
```

Comment:
This shouldn't pass the css linter. Mixins should go before declarations (at the top of a block)